app/models/food.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Food < ApplicationRecord
```

Comment:
**Optional**
- Well done with associations so far. However, i'd be much better for your models to write some validations for preventing bad users' behavior.

Reply:
<img width="419" alt="Captura de Pantalla 2022-08-06 a la(s) 17 26 28" src="https://user-images.githubusercontent.com/13943949/183257707-923e9873-29a1-4c35-a93d-4ee112977e3f.png">

Model validations added 👍